src/NuGet.Services.AzureSearch/Db2AzureSearch/Db2AzureSearchCommand.cs

```diff
@@ -133,17 +140,29 @@ private async Task InitializeAsync()
             await _blobContainerBuilder.CreateAsync(containerDeleted);
             await _indexBuilder.CreateSearchIndexAsync();
             await _indexBuilder.CreateHijackIndexAsync();
+
+            try
+            {
+                var storageResult = await _auxiliaryFileClient.LoadExcludedPackagesListAsync(etag: null);
+                _excludedPackagesList = storageResult.Data ?? new HashSet<string>();
```

Comment:
>_excludedPackagesList [](start = 16, length = 21)

This shouldn't be a field. It can be just be read in `ExecuteAsync` and passed as a parameter.